joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/VueJsDomAstCreationPassTest.scala

```diff
@@ -10,11 +10,17 @@ class VueJsDomAstCreationPassTest extends AbstractDomPassTest {
     "have correct structure vor simple vue.js template" in AstFixture(
       """
         |<template>
-        |<img v-for="image in images" :src="image.url" :alt="image.description" />
+        |<img v-for="image in images" :src="image.url" :attr="image.name" v-bind:alt="image.description" />
         |</template>""".stripMargin,
       "test.vue"
     ) { cpg =>
       cpg.file.name.l shouldBe List("test.vue")
+      cpg.templateDom.nameExact("JSXAttribute").code.l shouldBe List(
+        "v-for=\"image in images\"",
+        "src=\"image.url\"",
+        "attr=\"image.name\"",
```

Comment:
this doesn't seem to address the problem that we need to differentiate `:attr` and plain `attr`?

Reply:
JS with react does not allow `:` in front of its attributes so I have to strip them away. It was so before already but did not work for multiple attributes at one tag.

Reply:
Can we add the implicit `v-bind` then? The VueXss pass also needs to work for this `:` shortcut notation.

Reply:
I can't add code as this would mess up all code fields.

Reply:
Its replaced with whitespace. The only thing I can do there.

Reply:
Please note: it has been so already beforehand. The VueXss pass continues to work.